### PR TITLE
Write blog post announcing MUnit

### PR DESCRIPTION
**Pull request by olafurpg**
Live post https://olafurpg.github.io/munit/blog/2020/02/01/hello-world.html

### CODE REVIEW

**Comment by SethTisue** [comment]
https://www.youtube.com/watch?v=j83bGaauRXw

**Comment by olafurpg** [comment]
@SethTisue can you elaborate? :)

File: website/blog/2020-02-01-hello-world.md [hunk 1]
```diff
@@ -0,0 +1,270 @@
...
+## Tests as values
+
+If you know how to write normal Scala programs you should feel comfortable
+reasoning about how MUnit works.
+
+Internally, a core MUnit data structure is `GenericTest[T]`, which represents a
+single test case and is roughly defined like this.
+
+```scala
+case class GenericType[T](
```
Comment (gabro) [inline website/blog/2020-02-01-hello-world.md]: typo?

```suggestion
case class GenericTest[T](
```

File: website/blog/2020-02-01-hello-world.md [hunk 2]
```diff
@@ -0,0 +1,270 @@
...
+Check out the
+[generating test reports guide](https://scalameta.org/munit/docs/reports.html)
+to learn how to configure your build to upload test reports to Google Cloud
+using the MUnit sbt plugin. The plugin is implemented as an sbt `testsListener`
+so it works in theory with any testing library (including ScalaTest, utest, ...)
+although it has so far only been tested against MUnit.
+
+## Credits
+
+I want to thank to [@gabro](https://twitter.com/gabro27/) for implementing Dotty
```
Comment (gabro) [inline website/blog/2020-02-01-hello-world.md]: 🙏 

(minor typo? "to thank to")

**Comment by olafurpg** [comment]
@gabro thank you! 🙏 nice catch

**Comment by SethTisue** [comment]
@olafurpg I approve of your decision to publicize this

**Comment by olafurpg** [comment]
@SethTisue aah. Thanks 😄

**Pull request merged**